src/coprocessor/dag/expr/builtin_time.rs

```diff
@@ -1119,4 +1130,35 @@ mod tests {
             Datum::Time(Time::parse_utc_datetime("0000-00-00 00:00:00.000000", 6).unwrap()),
         );
     }
+
+    #[test]
+    fn test_to_days() {
+        let cases = vec![
```

Comment:
Can you add more test cases? e.g.`"0000-00-00"`, `"1992-13-00"`, `"2007-10-07 23:59:61"`, `123456789`

Reply:
It seems that `Time::parse_utc_datetime` cannot check invalid date such as `"1992-13-00"`. Do we need to implement the checking in `pub fn to_days()` or in `parse_datetime_internal()` which is in the file `mod.rs`

Reply:
Is this just the problem of the test?

Reply:
Yes, just the problem of test.

Reply:
Do we need to test these invalid cases? @huachaohuang

Reply:
I think it's fine. @GinYM if you are interested, please help to improve these DateTime tests in another PR :)

Reply:
@huachaohuang Sure, I will try it.